pkg/security/security.go

```diff
@@ -204,7 +212,8 @@ type SecretManager interface {
 
 // TokenExchanger provides common interfaces so that authentication providers could choose to implement their specific logic.
 type TokenExchanger interface {
-	ExchangeToken(context.Context, string, string) (string, time.Time, int, error)
+	ExchangeToken(context.Context, CredFetcher /*Credential fetcher*/, string, /*trustdomain*/
```

Comment:
You can just define the variable names in the interface instead of comments

Reply:
Done.